app/routes/_index.tsx

```diff
@@ -1,5 +1,8 @@
 import type { MetaFunction } from "@remix-run/node";
 
+import iconReactID from "~/assets/logo-reactjsid.svg";
```

Comment:
Make it consistent, just import `logo` is fine, as it's not an icon. Because the file name is `logo`.